src/components/navBar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from '@/components/navBar/navBar.module.scss';
+import Image from 'next/image';
+
+const RDSLogo = '/RDSLogo.png';
+
+const NavBar = () => (
+  <div>
```

Comment:
We don't need this `div` can we drop it?

Reply:
Yes we can drop it.